nil/api.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+from typing import List
+
+from loguru import logger
+
+from .interface import AbstractApi, AbstractDataStorage, AbstractHashStorage, Message, \
```

Comment:
Can we use a standard grouping mechanism for a long import list instead of string concatenation in the future?
```suggestion
from .interface import (AbstractApi, AbstractDataStorage, AbstractHashStorage, Message,
                                        UserMessage)
```
It is not a blocker, but it will be nice if we come to a style agreement here.

Reply:
Fixed. I see nothing bad with string concatenation though.

Reply:
*\*cough\** `isort` *\*cough\** 😃

Reply:
Sorted